### PR TITLE
Removed red shield in forwarding preview.

### PR DESCRIPTION
**Pull request by ankur12-1610**
Signed-off-by: ankur12-1610 <anknerd.12@gmail.com>

Fixes: [#19913](https://github.com/vector-im/element-web/issues/19913)

Before:
red-unsolved

After:
red-solved

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Removed red shield in forwarding preview. ([\#7447](https://github.com/matrix-org/matrix-react-sdk/pull/7447)). Contributed by @ankur12-1610.

Preview: https://61c6096ee6b48154630aae7f--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/dialogs/_ForwardDialog.scss [hunk 1]
```diff
@@ -54,6 +54,10 @@ limitations under the License.
             display: none;
         }
 
+        .mx_EventTile_e2eIcon_warning {
```
Comment (SimonBrandner) [inline res/css/views/dialogs/_ForwardDialog.scss:57]: While we're here, I think we can merge all the `display: none` into one selector, e.g.:

```css
.mx_EventTile_e2eIcon_unencrypted,
.mx_EventTile_e2eIcon_warning,
...
.mx_MFileBody_download {
	display: none
}
```

File: res/css/views/dialogs/_ForwardDialog.scss [hunk 2]
```diff
@@ -50,11 +50,9 @@ limitations under the License.
...
         // When forwarding messages from encrypted rooms, EventTile will complain
         // that our preview is unencrypted, which doesn't actually matter
-        .mx_EventTile_e2eIcon_unencrypted {
-            display: none;
-        }
-
         // We also hide download links to not encourage users to try interacting
+        .mx_EventTile_e2eIcon_unencrypted,
+        .mx_EventTile_e2eIcon_warning,
         .mx_MFileBody_download {
```
Comment (SimonBrandner) [inline res/css/views/dialogs/_ForwardDialog.scss:56]: `mx_EventTile_msgOption` as well, please :)

Comment (ankur12-1610) [reply res/css/views/dialogs/_ForwardDialog.scss:53]: > `mx_EventTile_msgOption` as well, please :)

Done :)

**Review by SimonBrandner — APPROVED** [review]
Thank you!

**Comment by ankur12-1610** [comment]
> Thank you!

Anytime :)

**Comment by ankur12-1610** [comment]
> Thank you!

@SimonBrandner there is still 1 pending review :sweat_smile: can you please request the review.

**Comment by SimonBrandner** [comment]
> @SimonBrandner there is still 1 pending review sweat_smile can you please request the review.

This is awaiting a review from a team member, which I am not, so you'll have to wait a bit

**Comment by ankur12-1610** [comment]
> > @SimonBrandner there is still 1 pending review sweat_smile can you please request the review.
> 
> This is awaiting a review from a team member, which I am not, so you'll have to wait a bit

Ah! okay np :)

**Review by t3chguy — APPROVED** [review]
Looks sane to me, thanks!

**Pull request merged**